src/my-archive/routes.ts

```diff
@@ -0,0 +1,27 @@
+import { Bookmarks } from './views/Bookmarks';
+import { Collections } from './views/Collections';
+import { Favourites } from './views/Favourites';
+import { Folders } from './views/Folders';
+
+export const MY_ARCHIVE_ROUTES = [
+	{
+		path: '/my-archive/collections',
```

Comment:
routes have to be in dutch from now on (decision by VIAA/Bart)
**mijn archief** will be called: **mijn werkruimte** (https://district01.atlassian.net/browse/AVO2-363)
lets make the internal variables: `my-workspace`